docs/tools_overview.md

```diff
@@ -207,3 +207,15 @@ output: T4 format data
 ```bash
 python -m perception_dataset.convert --config config/rosbag2_to_t4/convert_pseudolabel_lidar.yaml
 ```
+
+### Annotated T4 format to Deepen format
+
+In case you may want to modify the annotation of the T4 format data, you can also convert the annotated T4 format data to Deepen format.
+NOTE: By default the conversion script will convert sensor data and annotation data, but you may change `label_only` to `true` in the config file to convert only the annotation data.
+
+input: Annotated T4 format data
+output: deepen-format data
```

Comment:
Could it be like
```suggestion
output: deepen-format data (sensor data and label data)
```

Reply:
Updated: https://github.com/tier4/tier4_perception_dataset/pull/57/commits/589b6e08b61ddd363355f31af693a80be648561b